src/UnitTests/NullBehavior.cs

```diff
@@ -82,6 +84,12 @@ public void Should_return_null_for_nullable_properties()
             }
 
             [Fact]
+            public void Should_return_not_null_for_nullable_properties_that_are_user_defined()
+            {
+                _result.NullableMapFrom.ShouldEqual(0);

```

Comment:
I refer to line 49.  Cause it was user defined from MapFrom, ala a CustomExpression we use the MapFrom's expression's return type which is int.

Yea it's not the best example.  The issue is if I put conditions `? :` into line 49 the seal would throw an exception.  So I'm just saying if user defined it it's possible they are there and therefore we can't assume the nullable return type will work.  So tests just say it doesn't null on user defined.  Again not the best of tests.